mongo_orchestration/servers.py

```diff
@@ -308,8 +309,23 @@ def start(self, timeout=300):
                     "Server did not respond to 'isMaster' after %d attempts."
                     % timeout)
         except (OSError, TimeoutError):
-            logger.exception("Could not start Server.")
-            raise
+            logpath = self.cfg.get('logpath')
+            if logpath:
+                # Copy the server logs into the mongo-orchestration logs.
+                logger.error(
+                    "Could not start Server. Please find server log below.\n"
+                    "=====================================================")
+                with open(logpath) as lp:
+                    logger.error(lp.read())
```

Comment:
I'm a little worried that this could be a huge amount of data if the logfile isn't new for each run. Should I worry?